docs/case-studies/pop-health/index.rst

```diff
@@ -0,0 +1,38 @@
+====================================================================================================================
+The potential impact of declining development assistance for healthcare on population health: projections for Malawi
+====================================================================================================================
+
+*Margherita Molaro et al.*
```

Comment:
we've lost the hyperlink

https://www.medrxiv.org/content/10.1101/2024.10.11.24315287v1#:~:text=We%20estimate%20that%20the%20projected,percentage%20of%20GDP%20remains%20unchanged.

Reply:
Good catch, will add this in!